app/components/strata/flows/task_list_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<%# locals: (flow:, show_step_label:) %>
+
+<ul class="usa-collection">
+    <% @flow.tasks.each do |task| %>
+        <%= render @task_section_component_class.new(flow: @flow, task:, show_step_label: @show_step_label) %>
+    <% end %>
+</ul>
+
+<%= button_to t(".actions.review_and_submit"),
+            @flow.end_path,
+            method: :get,
+            class: "usa-button margin-top-2",
+            disabled: !@flow.completed? %>
```

Comment:
This maybe shouldn't be in the view component? Folks may want to render something different, or we may need to support different button text.